Description: Fake setuptools_scm
 This change creates the same file setuptools_scm would, so we dont get an error
 when running `clean`, because the file is already there and there's no diff
 with the source tarball. we may have to update it everytime there's a new release,
 which is not ideal, but for now it's a good stopgape.
Author: morph@debian.org
Origin: vendor
Forwarded: not-needed
Last-Update: 2021-10-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- /dev/null
+++ b/lib/matplotlib/_version.py
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+version = '3.5.0'
+version_tuple = (3, 5, 0)
